domains/runtime/eth-relay/src/runtime.rs

```diff
@@ -0,0 +1,539 @@
+use domain_runtime_primitives::opaque;
+pub use domain_runtime_primitives::{
+    AccountId, Address, Balance, BlockNumber, Hash, Index, RelayerId, Signature,
+};
+use frame_support::dispatch::DispatchClass;
+use frame_support::traits::{ConstU16, ConstU32, Everything};
+use frame_support::weights::constants::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight};
+use frame_support::weights::{ConstantMultiplier, IdentityFee, Weight};
+use frame_support::{construct_runtime, parameter_types};
+use frame_system::limits::{BlockLength, BlockWeights};
+use pallet_transporter::EndpointHandler;
+use sp_api::impl_runtime_apis;
+use sp_core::crypto::KeyTypeId;
+use sp_core::OpaqueMetadata;
+use sp_domains::DomainId;
+use sp_messenger::endpoint::{Endpoint, EndpointHandler as EndpointHandlerT, EndpointId};
+use sp_messenger::messages::{CrossDomainMessage, MessageId, RelayerMessagesWithStorageKey};
+use sp_runtime::traits::{AccountIdLookup, BlakeTwo256, Block as BlockT};
+use sp_runtime::transaction_validity::{TransactionSource, TransactionValidity};
+use sp_runtime::{create_runtime_str, generic, impl_opaque_keys, ApplyExtrinsicResult};
+pub use sp_runtime::{MultiAddress, Perbill, Permill};
+use sp_std::marker::PhantomData;
+use sp_std::prelude::*;
+#[cfg(feature = "std")]
+use sp_version::NativeVersion;
+use sp_version::RuntimeVersion;
+use subspace_runtime_primitives::{SHANNON, SSC};
+
+#[cfg(any(feature = "std", test))]
+pub use sp_runtime::BuildStorage;
```

Comment:
not sure if we are using this anywhere

Reply:
This was copied from `core-payments` runtime. 

Upon investigation, It is indeed not in use both here as well as in `core-payments`. Maybe it is worth removing from `core-payments`. I have removed it from here.